lib/engines/content_block_manager/app/components/content_block_manager/content_block_edition/details/fields/array_item_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if array_items["type"] == "string" %>
+  <%= render "govuk_publishing_components/components/input", {
```

Comment:
in a future PR, this could have something like the `component_for_field`, so that it can auto-render a component based on the type?

Reply:
Yeah, I think that would work, I just didn't want to get too much in the weeds for now